pkg/query/api/v1.go

```diff
@@ -204,6 +204,17 @@ func (api *API) parseEnableDedupParam(r *http.Request) (enableDeduplication bool
 	return enableDeduplication, nil
 }
 
+func (api *API) parseReplicaLabelsParam(r *http.Request) (replicaLabels []string, _ *ApiError) {
+	if err := r.ParseForm(); err != nil {
+		return nil, &ApiError{ErrorInternal, errors.Wrap(err, "parse form")}
+	}
+
+	if len(r.Form["replicaLabels[]"]) > 0 {
```

Comment:
Why not being consistent with every parse function:

```
const dedupParam = "dedup"
	enableDeduplication = true

	if val := r.FormValue(dedupParam); val != "" {
		var err error
		enableDeduplication, err = strconv.ParseBool(val)
		if err != nil {
			return false, &ApiError{errorBadData, errors.Wrapf(err, "'%s' parameter", dedupParam)}
		}
	}
	return enableDeduplication, nil
```
Essentially `ParseForm` vs `FormValue`?

Reply:
The golang API states that for multiple values need to use `ParseForm`

https://godoc.org/net/http#Request.FormValue

Reply:
fair. (: